src/composables/balances.ts

```diff
@@ -69,10 +68,9 @@ export function useBalances({ store }: IDefaultComposableOptions) {
   }
 
   async function updateBalances() {
-    const aeSdk = await getAeSdk();
     const balancesPromises = accounts.value.map(
       // TODO - type address in IAccount interface
-      ({ address }) => aeSdk.getBalance(address as Encoded.AccountAddress).catch((error) => {
+      ({ address, protocol }) => getProtocolSdk(protocol).getBalance(address).catch((error) => {
```

Comment:
By the way the comment above can be removed as it's already typed.

Reply:
Haha. Sure